### PR TITLE
build: run browserstack on circleci

### PR DESCRIPTION
**Pull request by DevVersion**

### CODE REVIEW

**Comment by DevVersion** [comment]
This is blocked for now because due to the GitHub issues, CircleCI is not able to run jobs for this PR.

File: .circleci/config.yml [hunk 1]
```diff
@@ -189,6 +208,7 @@ workflows:
   unit_tests:
     jobs:
       - tests_local_browsers
+      - browserstack_tests
```
Comment (josephperrott) [inline .circleci/config.yml]: Can we do `tests_browserstack`. So the naming convention is consistent

Comment (DevVersion) [reply .circleci/config.yml]: Good point. Will update that

**Comment by josephperrott** [comment]
CLA confirmed manually

**Pull request merged**